nodes/src/nodes/llm_openai_api/IGlobal.py

```
@@ -0,0 +1,95 @@
+            model = config.get('model')
+            base_url = config.get('base_url') or None
+            modelTotalTokens = config.get('modelTotalTokens')
+
+            # Only validate tokens > 0 if provided
+            if modelTotalTokens is not None and modelTotalTokens <= 0:
+                warning('Token limit must be greater than 0')
+                return
+
+            # Simple API validation using provider-driven exceptions
+            try:
+                kwargs = dict(api_key=apikey)
```

Ruff (0.15.7)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)